### ISSUE TITLE
Issue when attempting to load the generated svg as an Image()

### ISSUE THREAD

**Issue by HairMachine** [chartist-js/chartist#916]
Hi, thanks for this library.

We're looking to replace some existing charts with chartist. One of the features of the existing implementation (which is using d3.js I believe) is the ability to download a PDF of the chart, which currently works using jsPDF (https://github.com/MrRio/jsPDF). However, when generating the chart using this library the PDF download feature no longer works.

I've narrowed down what I think is happening to this: https://jsfiddle.net/8oxm059n/

Basically for some reason the svg produced by chartist doesn't seem to play happily with Image() here.

I've letft in a silly test SVG (commented out) - you can confirm that the basic code works for this, but not the svg produced by chartist.

Any ideas on what I can do about this? Is this approach just not going to work, can I change something in configuration or similar to fix it, or am I just making some terribly foolish error?

Thanks.

**Comment by HairMachine** [comment]
Have had a look around, and the issue seems closely related to this one: https://github.com/gionkunz/chartist-js/issues/164

Changing the xmlns:ct="http://gionkunz.github.com/chartist-js/ct to the standard xmlns="http://www.w3.org/2000/svg" solves the issue and allows the PDF to be downloaded.

**Comment by samuelgeojit** [comment]
@HairMachine  iam also lokking for  a feature to export as pdf can you pls help me in this topic
